flink-table/flink-table-planner/src/test/java/org/apache/flink/table/utils/TestUserClassLoaderJar.java

```diff
@@ -34,12 +34,13 @@
 import java.util.jar.JarEntry;
 import java.util.jar.JarOutputStream;
 
-/** Mainly used for testing classloading of UDF dependencies. */
+/** Mainly used for testing classloading. */
 public class TestUserClassLoaderJar {
 
-    private static final String GENERATED_UDF_CLASS = "LowerUDF";
+    /** Legacy code. */
+    public static final String GENERATED_UDF_CLASS = "LowerUDF";
 
-    private static final String GENERATED_UDF_CODE =
+    public static final String GENERATED_UDF_CODE =
```

Comment:
Move these back to `flink-sql-client`. They are used only by tests in SQL client.

Reply:
done